src/Tracing.cpp

```diff
@@ -6,11 +6,6 @@
 namespace Halide {
 namespace Internal {
 
-int tracing_level() {
```

Comment:
Might be good to still have something sniff for HL_TRACE and warn the user that this is now done via target features (at least for a little while)?